sdcm/utils/sstable/sstable_utils.py

```diff
@@ -216,6 +216,17 @@ def get_tombstone_date(tombstone_deletion_info: str) -> datetime.datetime:
         full_deletion_date_datetime = datetime.datetime.strptime(full_deletion_date, '%Y-%m-%d %H:%M:%S')
         return full_deletion_date_datetime
 
+    def corrupt_sstables(self, sstables_to_corrupt_count: int = 1):
```

Comment:
you have anything calling this ? (I guess it's for testing proposes
can you have a test integration that actually uses it ?

Reply:
open a task for unittesting this one

Reply:
https://github.com/scylladb/scylla-cluster-tests/issues/9909